pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -52,11 +52,14 @@ def __init__(
                 arch = "s390x"
             elif "ppc64le" in nightly:
                 arch = "ppc64le"
+            elif "arm64" in nightly:
+                arch = "aarch64"
             else:
                 arch = "x86_64"
             if ":" not in nightly:
-                # prepend pullsepc URL to nightly name
-                arch_suffix = "" if arch == "x86_64" else "-" + arch
+                # prepend pullspec URL to nightly name
+                # TODO: proper translation facility between brew and go arch nomenclature
+                arch_suffix = "" if arch == "x86_64" else "-amd64" if arch == "aarch64" else "-" + arch
```

Comment:
`arch_suffix = "" if arch == "x86_64" else "-arm64" if arch == "aarch64" else "-" + arch

iow `s/amd64/arm64/' in this line

Reply:
thanks, i must be blind

Reply:
```suggestion
                arch_suffix = "" if arch == "x86_64" else "-arm64" if arch == "aarch64" else "-" + arch
```